src/main/telemetry/smartport.c

```diff
@@ -365,9 +365,13 @@ static void processMspPacket(mspPacket_t* packet)
 {
     initSmartPortMspReply(0);
 
-    if (mspFcProcessCommand(packet, &smartPortMspReply, NULL) == MSP_RESULT_ERROR) {
+    mspPostProcessFnPtr mspFcProcessFn = NULL;
```

Comment:
I'd rename this guy to `mspPostProcessFn` to be consistent with the rest of the use of the PostProcess function pointer.